src/hotspot/cpu/aarch64/c1_FrameMap_aarch64.cpp

```diff
@@ -183,7 +183,7 @@ void FrameMap::initialize() {
   map_register(i, r17); r17_opr = LIR_OprFact::single_cpu(i); i++;
 #ifndef R18_RESERVED
   // See comment in register_aarch64.hpp
-  map_register(i, r18_tls); r18_opr = LIR_OprFact::single_cpu(i); i++;
+  map_register(i, r18_reserved); r18_opr = LIR_OprFact::single_cpu(i); i++;
```

Comment:
Upstream has `r18_tls` integrated https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/aarch64/c1_FrameMap_aarch64.cpp#L186, let's revert this?